guides/common/modules/proc_configuring-pull-based-transport-for-remote-execution.adoc

```diff
@@ -0,0 +1,17 @@
+[id="configuring-pull-based-transport-for-remote-execution_{context}"]
+= Configuring Pull-based Transport for Remote Execution
+
+Use this procedure to set up `pull-mqtt` mode for pull-based remote execution transport.
+
+.Procedure
+ifdef::satellite,orcharhino,foreman[]
+* Ensure your machine certificates are installed.
+** If you use Puppet, you can use Puppet certificates.
+* Install `yggdrasil` and `foreman_ygg_worker`.
+* Configure `yggdrasil`.
+* Start `yggdrasil`.
+endif::[]
+ifdef::katello[]
```

Comment:
This is somewhat tricky. `katello-pull-transport-migrate` script is useable only when katello is available (ie. katello, orcharhino, satellite) and when the target hosts have valid entitlement certs.

Also, where possible, it would probably better to use the `katello-pull-transport-migrate` package from client repositories, rather than pointing people at github.